src/main/kotlin/it/pagopa/aca/domain/Iupd.kt

```diff
@@ -0,0 +1,15 @@
+package it.pagopa.aca.domain
+
+@JvmRecord
```

Comment:
why a java record here? kotlin data class should just be good here

Reply:
https://kotlinlang.org/docs/jvm-records.html#requirements 🙂 tell me what you think

Reply:
Yes, my question is, why use a java record? a kotlin [data class](https://kotlinlang.org/docs/data-classes.html) should model this entity